sdk/communication/Azure.Communication.Common/api/Azure.Communication.Common.netstandard2.0.cs

```diff
@@ -9,6 +9,14 @@ public abstract partial class CommunicationIdentifier
     {
         protected CommunicationIdentifier() { }
     }
+    public sealed partial class CommunicationTokenCredential : System.IDisposable
+    {
+        public CommunicationTokenCredential(Azure.Communication.Identity.CommunicationTokenRefreshOptions tokenRefreshOptions) { }
```

Comment:
I think we should move every type out of Azure.Communication.Identity up to Azure.Communication. Essentially get rid of the extra folder.

Reply:
That's to follow the guideline to avoid sub namespaces with only a few types. When in doubt let's ask Christopher.

Reply:
Done, thanks Dom